### PR TITLE
added text for PR style cep

### PR DESCRIPTION
**Pull request by gidden**
A suggested style guide for code base pull requests

### CODE REVIEW

File: source/cep/cep0.rst [hunk 1]
```diff
@@ -47,6 +47,7 @@ Index
     PA - 3 - Release Procedure - Scopatz <cep3>
     PD - 4 - Attribution of Code Products in the |Cyclus| Ecosystem - Huff <cep4>
     PL - 5 - Archetype Development Best Practices - Gidden <cep5>
+    PD - 5 - |Cyclus| Pull Request Style Guide - Gidden <cep6>
```
Comment (gonuke) [inline source/cep/cep0.rst]: Probable should be CEP 6 here

File: source/cep/cep6.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+:Type: Process
+:Created: 2015-05-04
+
+Abstract
+========
+
+All accepted pull requests must have a reasonable description of their
+purpose. Requiring a consistent format across projects for each pull request
+description assists in their review and allows for certain summary automation.
+
```
Comment (gonuke) [inline source/cep/cep6.rst]: I wonder if it makes sense to insert more on motivation here - the motivation of automation is compelling and makes me more interested in reading on to adopt this.

Comment (gidden) [inline source/cep/cep6.rst]: There's a section on automation at the end. You're requesting more about it
in the abstract, correct?

On Tue, May 5, 2015 at 2:06 PM, Paul Wilson <notifications@github.com>
wrote:

> In source/cep/cep6.rst
> <https://github.com/cyclus/cyclus.github.com/pull/134#discussion_r29701690>
> :
>
> > +
> > +:CEP: 6
> > +:Title: |Cyclus| Pull Request Style Guide
> > +:Last-Modified: 2015-05-04
> > +:Author: Matthew Gidden
> > +:Status: Draft
> > +:Type: Process
> > +:Created: 2015-05-04
> > +
> > +Abstract
> > +========
> > +
> > +All accepted pull requests must have a reasonable description of their
> > +purpose. Requiring a consistent format across projects for each pull request
> > +description assists in their review and allows for certain summary automation.
> > +
>
> I wonder if it makes sense to insert more on motivation here - the
> motivation of automation is compelling and makes me more interested in
> reading on to adopt this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/134/files#r29701690>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

File: source/cep/cep6.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Overview
+========
+
+The goal of this document is to define a minimum uniform style for pull
+requests. Having a uniform style will aide in the timely review of PRs as all
```
Comment (opotowsky) [inline source/cep/cep6.rst]: nitpick: using an acronym introduction "pull requests (PRs)". This isn't super important for the usual audience but possibly helpful to a new person.

File: source/cep/cep6.rst [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+reviewers can expect standard information. Furthermore, given a uniform style,
+automated tools can be developed that assist in other managerial tasks. 
+
+Github supports it's own version of a flavored markdown `syntax`_. That syntax
```
Comment (opotowsky) [inline source/cep/cep6.rst]: grammar: its

**Comment by opotowsky** [comment]
Thanks for doing this @gidden! I find it quite helpful. I can merge but perhaps others should have a look since it's a CEP?

File: source/cep/cep6.rst (hunk 2 above)
Comment (gonuke) [inline source/cep/cep6.rst]: Yep - I found myself looking for more motivation about this

**Comment by gidden** [comment]
updated

**Comment by gidden** [comment]
@opotowsky, we can probably just wait on this PR until the CEP is accepted (unless others disagree).

**Comment by gidden** [comment]
Since this has received a lot of discussion already on the list and here, I am going to merge, noting that it is still designated as a draft.

**Pull request merged**